specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -2230,6 +2230,11 @@
         "tags": [
           "Providers"
         ],
+        "x-ms-examples": {
+          "Get a resource provider at tenant scope": {
+            "$ref": "./examples/GetNamedProviderAtTenant.json"
```

Comment:
does the change only apply to tenant level providers call?

Reply:
It applies to all providers/ calls. I can add example for each individual flavor if you think it's necessary.